lib/config/fileconf.go

```diff
@@ -855,6 +855,9 @@ type PAM struct {
 	// UsePAMAuth specifies whether to trigger the "auth" PAM modules from the
 	// policy.
 	UsePAMAuth bool `yaml:"use_pam_auth"`
+
+	// Override specifies if local configuration will override cluster PAM configuration.
+	Override bool
```

Comment:
The rest of the fields have yaml tags attached. If this field is not exposed in configuration - use an explicit exclude tag value.